src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -142,6 +143,18 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ project, en
     );
   };
 
+  const CreateWorkPackageButton = () => (
+    <MenuItem
+      onClick={() => history.push(`${routes.CHANGE_REQUESTS_NEW}?wbsNum=${projectWbsPipe(project.wbsNum)}&createWP=${true}`)}
```

Comment:
can you make this a helper method and pass it in, since the logic of constructing the URL is a bit involved